src/module/actor/data-model-character.js

```diff
@@ -25,7 +25,8 @@ export default class OseDataModelCharacter extends foundry.abstract.TypeDataMode
           "significantTreasure"
         ),
         scores: this.scores,
```

Comment:
@arcos This should give you access to the strength mod. The original intent of this object was to give someone who implements their own encumbrance scheme whatever modifiers they might need from the character.